airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/api.py

```diff
@@ -158,10 +159,15 @@ def call(
 class API:
     """Simple wrapper around Facebook API"""
 
+    API_VERSION = "v14.0"
```

Comment:
this is still pointing to v14 because I want the acceptance tests to pass before I upgrade to v15...